### ISSUE TITLE
[request] automatic file names

### ISSUE THREAD

**Issue by costela** [goreleaser/nfpm#70]
Hi,

nfpm is great, but maybe it would be even greater if it supported automatically naming the generated files.
At least in the `deb` context, the filename usually has a [standardized structure](https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics#s-pkgname), roughtly:
```
{{ name }}_{{ version }}_{{ arch }}.deb
``` 
Where `name` is lowercased and basically piped through `s/[^a-z]/-/g`.

This would make it easier to use generated files with various Debian/Ubuntu package handling tools that assume this format is used.

I'm unfortunately not sure if there's an equivalent convention on the RPM side.

Maybe allow passing `--target deb` as a sign that the filename should be generated? Or if that's too implicit, a separate flag?

I'm aware that the feature overlaps with what goreleaser's `name_template`, but IMHO this would still be useful as a feature in `nfpm` itself.

**Comment by erikgeiser** [comment]
I think this can be closed now after #157 was merged. I did not notice this issue when I created the pull request so I didn't reference it.

**Comment by caarlos0** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
